FIPS related docs (#17196)
keycloak/keycloak: docs/guides/src/main/server/keycloak-truststore.adoc

* `WILDCARD` allows wildcards in subdomain names, such as *.foo.com.
* When using `STRICT`, the Common Name (CN) must match the hostname exactly.

type::

> The type of truststore like for example `jks`, `pkcs12` or `bcfks`. If not provided, the type would be detected based on the truststore

Reviewer: ```suggestion
The type of truststore, such as `jks`, `pkcs12` or `bcfks`. If not provided, the type would be detected based on the truststore
```
